src/axom/sidre/core/View.hpp

```diff
@@ -1370,6 +1370,18 @@ class View
    */
   void copyView(View* copy) const;
 
+  /*!
+   * \brief Deep copy view contents into an undescribed EMPTY view.
```

Comment:
Please state "contents of this view".  The explicit "this view" makes things clearer as a user glances through the docs.